framework/src/media/audio/resample/remix.c

```diff
@@ -0,0 +1,293 @@
+/****************************************************************************
```

Comment:
How move this file to mediaUtils?
Of course, not this PR

Reply:
yes, I can do it in this PR :)